### DISCUSSION TITLE
Configuration problem  - data directory is required\

### DISCUSSION THREAD

**Discussion by dtwarogpl** [mxpv/podsync#364] [General]
Hello, Im trying to run podsync from docker container. Have tried almost everything but Im, not able to. 

I get same error every time:
`time="2022-07-25T12:45:54Z" level=fatal msg="failed to load configuration file" error="1 error occurred:\n\t* data directory is required\n\n"`

My docker-compose file:
`version: '2.2'
services:
  podsync:
    container_name: podsync
    image: mxpv/podsync:latest
    restart: always
    ports:
      - 8081:80
    volumes:
      - ./data:/app/data/
      - ./config.toml:/app/config.toml
`

my configuration:
`# Configure where to store the episode data
[storage]
  type = "local"
  [storage.local]
  data_dir = "/app/data"
[tokens]
youtube = "AIzaSyARXXXXXXXXXXX"

[feeds]
    [feeds.ID1]
    url = "https://www.youtube.com/channel/UCIXXXXX"
    page_size = 5
    format = "audio"
    clean = { keep_last = 10 }`

Appreciate any help

**Comment by Contextualist** [comment]
The config example is consistent with the developing branch. If you are using the Docker container, i.e. latest released version, use the config like below:
```toml
[server]
port = 8080
data_dir = "/data/podsync/"

# ... the rest are the same, except that you don't need the `storage` section
```

The above applies until the next version is released.
